-Failed tests
Test 1: Teste 1
Test 2: Teste 2
-Test 1: Teste 1 (-10.000)
Incorrect program result
 --- Input ---
> 3
>3
>6
>6
>8
>6
>7
>4
>3
>8

 --- Program output ---
>digite o tam da matriz:digite o el:Traceback (most recent call last):
>  File "av2_p3_m2.py", line 53, in <module>
>    n=n(w)
>  File "av2_p3_m2.py", line 7, in n
>    for i in range (0,b.shape[0],1):
>NameError: name 'b' is not defined

 --- Expected output (text)---
> '1'
>'7'

-Test 2: Teste 2 (-10.000)
Incorrect program result
 --- Input ---
> 4
>16
>3
>2
>13
>5
>10
>11
>8
>8
>6
>7
>12
>4
>15
>14
>1

 --- Program output ---
>digite o tam da matriz:digite o el:Traceback (most recent call last):
>  File "av2_p3_m2.py", line 53, in <module>
>    n=n(w)
>  File "av2_p3_m2.py", line 7, in n
>    for i in range (0,b.shape[0],1):
>NameError: name 'b' is not defined

 --- Expected output (text)---
> '9'
>'8'

-Summary of tests
>+------------------------------+
>|  2 tests run/ 0 tests passed |
>+------------------------------+

